initializers/logger.js

```diff
@@ -32,7 +32,7 @@ const createGcLogger = logger => ({
 
 			if(typeof first === 'string') {
 				message = first;
-			} else if (first instanceof Error) {
+			} else if (first && first.message) {
 				err = first;
 				message = data.supplementalMessage || err.message;
 			} else {
```

Comment:
probably update the error message below too, then (for some reason it's not letting me comment on that specific line)

Reply:
scratch that, probably not a good idea as discussed